feature-libs/quote/components/details/comment/quote-details-comment.component.ts

```diff
@@ -130,7 +130,7 @@ export class QuoteDetailsCommentComponent {
     return this.quoteDetails$.pipe(
       map((quote) => {
         const itemList: Item[] = [{ id: ALL_PRODUCTS_ID, name: allProducts }];
-        quote.entries?.forEach((entry) => {
+        quote?.entries?.forEach((entry: OrderEntry) => {
```

Comment:
```suggestion
        quote.entries?.forEach((entry: OrderEntry) => {
```
also the following chnages

Reply:
Done